sites/public/src/components/listing/ListingView.tsx

```diff
@@ -595,19 +600,20 @@ export const ListingView = (props: ListingProps) => {
       <div className="w-full md:w-2/3 md:mt-3 md:hidden md:mx-3 border-gray-400 border-b">
         <ApplicationStatus content={appStatusContent} subContent={appStatusSubContent} />
         <div className="mx-4">
-          <DownloadLotteryResults
+          {/* TODO: same here about events */}
+          {/* <DownloadLotteryResults
             resultsDate={dayjs(lotteryResults?.startTime).format("MMMM D, YYYY")}
```

Comment:
are we missing another dto export like events was or is something else up with these?

Reply:
Good catch, I forgot to go through my TODOs after our conversation. This has been updated